docs/getting_started.md

```diff
@@ -71,6 +81,41 @@ Additional flags for cluster configuration exist but are typically not needed. C
 * _[Example Kubernetes manifest](../kube/manifests/prometheus.yaml)_
 * _[Example Kubernetes manifest with GCS upload](../kube/manifests/prometheus-gcs.yaml)_
 
+### Query Access
+
+Thanos comes with a highly efficient gRPC-based Store API for metric data access across all its components. The sidecar implements it in front of its connected Prometheus server. While it is ready to use with the above example, we must additionally configure the sidecar to join a Thanos cluster.
+
+Let's extend the above sidecars to expose their gRPC Store API so that we can query metrics from both Prometheus instances.
+
+```
+thanos sidecar \
```

Comment:
I don't think duplication will help here.. can we revert this and make sure gossip part is explained:
- WHY we need gossip
- how to use it (your modified sidecar examples makes sense just single sidecar + query would be just fine)
- What are alternatives (--store  flag)